core/src/test/java/hudson/PluginWrapperTest.java

```diff
@@ -10,36 +10,35 @@
 import java.util.jar.Manifest;
 
 import jenkins.model.Jenkins;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Importing necessary classes is preferred

Reply:
Thanks, removed the star import